python/prophet/models.py

```diff
@@ -63,28 +65,16 @@ class CmdStanPyBackend(IStanBackend):
     def __init__(self):
         super().__init__()
         import cmdstanpy
-        cmdstanpy.set_cmdstan_path(
-            pkg_resources.resource_filename("prophet", f"stan_model/cmdstan-{self.CMDSTAN_VERSION}")
-        )
+        local_cmdstan = pkg_resources.resource_filename("prophet", f"stan_model/cmdstan-{self.CMDSTAN_VERSION}")
+        if os.path.exists(local_cmdstan):
+            cmdstanpy.set_cmdstan_path(local_cmdstan)
 
     @staticmethod
     def get_type():
         return StanBackendEnum.CMDSTANPY.name
 
-    def _add_tbb_to_path(self):
```

Comment:
Just wondering if you've tested the pip Windows install on your machine (you can download the wheel)? I added this because I was getting an error about TBB when trying to load the model.

Reply:
I was waiting for the other test issues to be resolved before trying on my windows machine. Doing that now.

Reply:
I believe I've located the windows issue